### DISCUSSION TITLE
How to enable Tag "0"?

### DISCUSSION THREAD

**Discussion by Zhou-Yicheng** [leftwm/leftwm#1149] [Q&A] [answered]
Is it possible to enable tag "0"?

**Comment by blackdragon2447** [comment, marked as answer]
Add `"0"` to the tags array in `~/.config/leftwm/config.run` and add keybinds to swith to the tag and to send windows to it.
example:
```
tags: [
...
     "8",
     "9",
     "0",
],
keybind: [
...
        (command: GotoTag, value: "10", modifier: ["modkey"], key: "0"),
        (command: MoveToTag, value: "10", modifier: ["modkey", "Shift"], key: "0"),
...
]
```
(the keybind value is the index of the tag in the tags array (starting at 1)).

**Comment by Zhou-Yicheng** [comment]
It works! thx
